featurebyte/session/snowflake.py

```diff
@@ -385,10 +376,5 @@ async def drop_all_objects_in_working_schema(self) -> None:
             ):
                 await self._drop_object("PROCEDURE", func_name_with_args)
 
-        await self._drop_tasks()
```

Comment:
This is no longer required since tasks are now scheduled outside of the warehouse.